dev_guide/jobs.adoc

```diff
@@ -125,3 +125,12 @@ The following example shows the part of a `*Job*` specifying
     activeDeadlineSeconds: 1800
 ----
 ====
+
+[[job-failure-policy]]
+== Job Backoff Failure Policy
+A Job can be considered a failed,  after a set amount of retries due to a
```

Comment:
Maybe "A job can be considered to be failed after a set amount..." Not sure if "failed" is a specific term and needs to be marked up or whatever?